src/acceptance/administrator/components/com_users/UserListCest.php

```diff
@@ -131,9 +131,9 @@ protected function toggleSendMail($I)
 		$I->amOnPage('/administrator/index.php?option=com_config');
 		$I->waitForText('Global Configuration', TIMEOUT, ['css' => '.page-title']);
 		$I->comment('I open the Server Tab');
-		$I->click(['link' => 'Server']);
+		$I->click('Server');
 		$I->comment('I wait for error reporting dropdown');
-		$I->click(['xpath' => "//input[@type='radio' and @value=0 and @name='jform[mailonline]']"]);
+		$I->click(['xpath' => "//input[@id='jform_mailonline1']"]);
```

Comment:
* Avoid XPath
* Please extract to PageObject

Reply:
I had to revert this part as tests failed anyhow